src/Symfony/Component/Messenger/Bridge/Amqp/CHANGELOG.md

```diff
@@ -6,3 +6,4 @@ CHANGELOG
 
  * Introduced the AMQP bridge.
  * Deprecated use of invalid options
+ * Add option to confirm message delivery
```

Comment:
can you please move this to a new 5.2 section?

Reply:
Sure! ;-)